### ISSUE TITLE
Update fluent-plugin-kubernetes_metadata_filter to >= 2.5.1

### ISSUE THREAD

**Issue by jjo** [bitnami/bitnami-docker-fluentd#11]
**Description**

Latest image (`bitnami/fluentd:1.11.1-debian-10-r33` at the time of this writing) ships with fluent-plugin-kubernetes==2.5.0,
which lacks 
**Steps to reproduce the issue:**

1. Have `bitnami/fluentd:1.11.1-debian-10-r33` or ealier running in your Kubernetes cluster (usually as a daemonset watching nodes' logs)
2. In a busy cluster, find them sometimes sporadically with:
```
#<Thread:0x0000564f417e6968@/opt/bitnami/fluentd/gems/fluent-plugin-kubernetes_metadata_filter-2.4.5/lib/fluent/plugin/filter_kubernetes_metadata.rb:277 run> terminated with exception (report_on_exception is true):
/opt/bitnami/fluentd/gems/fluent-plugin-kubernetes_metadata_filter-2.4.5/lib/fluent/plugin/kubernetes_metadata_watch_namespaces.rb:43:in `rescue in set_up_namespace_thread': undefined method `<' for nil:NilClass (NoMethodError)
	from /opt/bitnami/fluentd/gems/fluent-plugin-kubernetes_metadata_filter-2.4.5/lib/fluent/plugin/kubernetes_metadata_watch_namespaces.rb:38:in `set_up_namespace_thread'
	from /opt/bitnami/fluentd/gems/fluent-plugin-kubernetes_metadata_filter-2.4.5/lib/fluent/plugin/filter_kubernetes_metadata.rb:277:in `block in configure'
/opt/bitnami/fluentd/gems/fluent-plugin-kubernetes_metadata_filter-2.4.5/lib/fluent/plugin/kubernetes_metadata_watch_namespaces.rb:127:in `block in process_namespace_watcher_notices': Error while watching namespaces: too old resource version: 80400208 (80402691) (RuntimeError)

```
**Describe the results you received:**

Sporadically crashing fluentd Pods

**Additional information you deem important (e.g. issue happens only occasionally):**

Reported and fixed at https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter/issues/226,
specifically https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter/issues/226#issuecomment-656873024

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**

Kubernetes 1.16 using https://github.com/bitnami/kube-prod-runtime/blob/v1.6.1/manifests/components/images.json#L11

**Comment by carrodher** [comment]
Thanks, we will take a look at the plugins included in the image and release a new version updating them

**Comment by carrodher** [comment]
We just released a new version (`1.11.1-debian-10-r34`) and the corresponding rolling tags. Can you try this new revision? That is what I obtain from a running container:

```console
$ ls /opt/bitnami/fluentd/gems/ | grep kubernetes
fluent-plugin-kubernetes_metadata_filter-2.5.2
```

**Comment by jjo** [comment]
No such log traces observed during ~1week run in any of our internal clusters, thanks!
(will reopen if we observe any)
